### PR TITLE
add endpoint: get news by slug

### PR DESCRIPTION
**Pull request by khihadysucahyo**
## Changes
- add endpoint: get news by slug
- `ALTER table news modify slug varchar(100) UNIQUE NOT NULL;`

@jabardigitalservice/jds-backend 

## Evidence
project: Portal Jabar
title: Menambahkan endpoint get berita by slug
participants: @khihadysucahyo @rachadiannovansyah @firmanJS @ayocodingit

### CODE REVIEW

**Review by firmanJS — APPROVED** [review]

File: core-service/src/database/migrations/1629281242_initialize_schema.up.sql [hunk 1]
```diff
@@ -109,7 +109,7 @@ CREATE TABLE news (
   title varchar(80) NOT NULL,
   excerpt text NOT NULL,
   content text NOT NULL,
-  slug varchar(100) DEFAULT NULL,
+  slug varchar(100) UNIQUE NOT NULL,
```
Comment (ayocodingit) [inline core-service/src/database/migrations/1629281242_initialize_schema.up.sql:112]: Index?

Comment (khihadysucahyo) [reply core-service/src/database/migrations/1629281242_initialize_schema.up.sql:112]: ohiya ketinggalan (swipe right)

**Review by rachadiannovansyah — APPROVED** [review]
let's goo

File: core-service/src/database/migrations/1629281242_initialize_schema.up.sql (hunk 1 above)
Comment (khihadysucahyo) [reply core-service/src/database/migrations/1629281242_initialize_schema.up.sql:112]: dah ditambah man

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**